src/core/AudioParam.cpp

```diff
@@ -48,13 +47,13 @@ float AudioParam::value(ContextRenderLock& r)
     // Update value for timeline.
     if (r.context()) {
         bool hasValue;
-        float timelineValue = m_timeline.valueForContextTime(r, narrowPrecisionToFloat(m_value), hasValue);
```

Comment:
narrowPrecisionToFloat something to remove from WTF? ... these sorts of things, in a future rev I think, should be replaced by the equivalent of a gsl::narrow_cast to capture the narrowing properly.

Reply:
I saw the file was removed. Still like the notion of gsl::narrow_cast. Not gsl per se because no new dependencies...